chain/tendermint/src/protobuf/gogoproto.rs

```diff
@@ -0,0 +1 @@
+
```

Comment:
As we discussed, we should add this file and `chain/tendermint/src/protobuf/google.protobuf.rs` into `.gitignore`